### PR TITLE
Handle upstream timeout and return appropriate status code

### PR DESCRIPTION
**Pull request by lmineiro**

### CODE REVIEW

File: handlers/tokeninfo/proxy/handler.go [hunk 1]
```diff
@@ -93,6 +93,15 @@ func (h *tokenInfoProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Reque
 		upstreamTimer.UpdateSince(upstreamStart)
 		return nil
 	}, nil)
+
+	if err == hystrix.ErrTimeout {
```
Comment (hjacobs) [inline handlers/tokeninfo/proxy/handler.go]: what about other errors?

Comment (lmineiro) [inline handlers/tokeninfo/proxy/handler.go]: I'll add some more error/status code mappings to this pr

**Comment by lmineiro** [comment]
Feedback on the suggested error codes is welcome

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**